dotnet/samples/Hello/Hello.AppHost/Hello.AppHost.csproj

```diff
@@ -1,7 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
+  <Sdk Name="Aspire.AppHost.Sdk" Version="9.0.0" />
```

Comment:
It's something new to me, what's it for

Reply:
https://learn.microsoft.com/en-us/dotnet/aspire/get-started/aspire-overview